packages/workflow/src/Workflow.ts

```diff
@@ -1009,7 +1009,7 @@ export class Workflow {
 		nodeExecuteFunctions: INodeExecuteFunctions,
 		mode: WorkflowExecuteMode,
 		activation: WorkflowActivateMode,
-		isTest?: boolean,
+		{ isTest }: { isTest?: boolean } = {},
```

Comment:
we should either move more arguments into this object, or we should revert this change. having this just for `isTest` seems unnecessary.
or, maybe we could move `isTest` into `IWebhookData` ?

Reply:
Thanks! I did this to avoid passing standalone booleans into the call - this arg used to be more readable as `this.testWebhooks`, until we merged `ActiveWebhooks.ts` into this file as part of the review.

I find `{ isTest: true }` more readable that keeping a standalone boolean. While I agree this legacy code needs much more work and we should consolidate options into a single arg, I wonder if that should that be done later down the line? The goal of this PR is prepping test webhooks so I can make the changes for the multi-main setup we discussed on Friday.

Reply:
I generally agree, but the `isTest` changes in this PR seem like they are adding tech-debt. How about [something like this](https://github.com/n8n-io/n8n/commit/60c8a7f18e13797e5b34a4347fa8afe6ae321066)?

Reply:
Ah I see what you mean now! If you can please bring this in.